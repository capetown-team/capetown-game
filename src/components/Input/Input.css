.Input {
    width: 280px;
    height: 38px;
    vertical-align: middle;
    background: #F3F4F7;
    display: block;
    margin: auto;
    margin-top: 10px;
}