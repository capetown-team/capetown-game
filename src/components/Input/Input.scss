@import '../../styles/layout/colors';
.Input {
    width: 280px;
    height: 38px;
    background: $white;
    display: block;
    margin: auto;
}