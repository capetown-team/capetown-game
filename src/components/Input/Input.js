import React from 'react';

import './Input.css';

export class Input extends React.Component {
    render() {
        const { id, placeHolder } = this.props;
        console.log('input', this.props);
        return (          
            <input className="Input" placeholder={placeHolder} id={id}>
                
            </input>
        );
    }
}
export default Input;