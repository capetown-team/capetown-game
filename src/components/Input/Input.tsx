import React from 'react';

import './Input.scss';

interface InputProps {
    id?: string;
    placeHolder?: string;
  }
 
interface InputState {
}

export class Input extends React.Component<InputProps, InputState> {
    render() {
        const { id, placeHolder } = this.props;
        console.log('input', this.props);
        return (         
            <input className="Input" placeholder={placeHolder} id={id}>
                
            </input>
        );
    }
}