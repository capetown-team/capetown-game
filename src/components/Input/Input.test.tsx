import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import { Input } from './Input';

describe('<Input />', () => {
  it('should renders correct <Input />', () => {
    const wrapper = shallow(<Input>test</Input>);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
