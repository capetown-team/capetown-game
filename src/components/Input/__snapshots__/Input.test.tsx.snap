// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should renders correct <Input /> 1`] = `
<input
  className="input"
/>
`;
